app/src/pages/inside/projectSettingsPageContainer/content/elements/ruleList/propTypes.js

```diff
@@ -1,11 +1,13 @@
 import PropTypes from 'prop-types';
+import { ruleField } from 'pages/inside/projectSettingsPageContainer/content/notifications/propTypes';
 
 export const ruleListPropTypes = {
   items: PropTypes.array,
   actions: PropTypes.array,
   onToggle: PropTypes.func,
   disabled: PropTypes.bool,
   ruleItemContent: PropTypes.elementType,
+  ruleItemContentProps: PropTypes.arrayOf(ruleField),
```

Comment:
The `RuleList` component is widely used within settings (not only for notifications), so we need to have a more abstract type here, like ` PropTypes.object`.